libraries/fof/controller/controller.php

```diff
@@ -3347,56 +3347,15 @@ protected function _csrfProtection()
 				break;
 		}
 
-		$hasToken = false;
-		$session  = JFactory::getSession();
+		$token = new \Joomla\Cms\Session\CsrfToken(JFactory::getSession());
```

Comment:
FOF 2 is already EOL. So any temporary changes we make to it are there until it's completely removed from Joomla! 4. Don't sweat over it.